tests/functional/upgrade/test_noobaa.py

```
@@ -37,6 +42,7 @@ def test_fill_bucket(
     Test multi-region bucket creation using the S3 SDK. Fill the bucket for
     upgrade testing.
     """
+    global DOWNLOADED_OBJS
```

Ruff (0.15.13)

[warning] Using the global statement to update `DOWNLOADED_OBJS` is discouraged

(PLW0603)